include/solvers/multiphysics_interface.h

```diff
@@ -809,7 +904,7 @@ class MultiphysicsInterface
       }
   };
 
-
+  bool constraing_solver;
```

Comment:
What is this parameter and why is it public?

Reply:
This has been removes it was use in an other architecture that wasn't working